cves/kernel/CVE-2021-38300.yml

```diff
@@ -411,8 +426,8 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: This issue is mitigated by setting the flag sysctlnet.core.bpf_jit_enable to 0, which is the default.
```

Comment:
In this context, I think `secure_by_default` means to make sure all things are secure in the first place, so I would just reword the ending a little bit to say something along the lines of "which ensures correct functionality"